frontend/src/scenes/experiments/ExperimentForm.tsx

```diff
@@ -48,6 +50,34 @@ const StepInfo = (): JSX.Element => {
                         />
                     </LemonField>
                 </div>
+                {webExperimentsAvailable && (
+                    <div className="mt-10">
+                        <h3 className="mb-1">Experiment type</h3>
+                        <div className="text-xs text-muted">
+                            Select your experiment setup, this cannot be changed once saved.
+                        </div>
+                        <LemonDivider />
+                        <LemonRadio
+                            value={experiment.type}
+                            onChange={(type) => {
+                                setExperimentType(type)
+                            }}
+                            options={[
+                                {
+                                    value: 'product',
+                                    label: 'Product experiment',
+                                    description: 'Use custom code to manage how variants modify your product.',
+                                },
+                                {
+                                    value: 'web',
+                                    label: 'No-code web experiment',
+                                    description:
+                                        'Define variants on your website effortlessly using the PostHog toolbar, no coding required.',
```

Comment:
This is just my preference, but the word "effortlessly" sounds a bit marketing-y to me (there clearly _is_ some effort required), so I'd remove it :)

```suggestion
                                        'Define variants on your website using the PostHog toolbar, no coding required.',
```